terracotta-kit/pom.xml

```diff
@@ -135,7 +135,7 @@
                     <finalName>terracotta-${project.version}</finalName>
                     <appendAssemblyId>false</appendAssemblyId>
                     <attach>true</attach>
-                    <tarLongFileMode>gnu</tarLongFileMode>
```

Comment:
This change breaks packaging on my local machine

Reply:
Got checked-in by mistake. Will remove.